godtools/fi.lproj/Localizable.strings

```diff
@@ -0,0 +1,277 @@
+"language_onboard" = "80+ Languages";
+
+/* This is to indicate to the user that there are no additional tools or resources available for download because they have already downloaded everything. */
+"no_more_tools" = "You have downloaded all available tools.";
```

Comment:
I assume the translations will get loaded to these files when the Onesky download command is run in GHA?

Reply:
Hey @reldredge71 yeah I'm thinking these will get overwritten during the OneSky download.  Xcode initially creates them based off of the Base Localizable.strings.